src/vhost.c

```diff
@@ -162,9 +165,15 @@ static int vhost_poll(struct pg_brick *brick, uint16_t *pkts_cnt,
 
 	pkts_mask = pg_mask_firsts(count);
 	ret = pg_brick_side_forward(s, state->output,
-				 state->in, pkts_mask, errp);
+				    state->in, pkts_mask, errp);
 	pg_packets_free(state->in, pkts_mask);
 
+	/* count rx bytes: burst is packed so we can directly iterate */
+	in = state->in;
+	for (int i = 0; i < count; i++)
+		rx_bytes += rte_pktmbuf_pkt_len(in[i]);
```

Comment:
if you have a vtep on the way, packets size will increase and the result wont be the same